client/src/components/Nav/Nav.jsx

```diff
@@ -1,11 +1,47 @@
 import React, { Component } from 'react';
+import './hack-navbar.css';
 
 class Nav extends Component {
-   render() {
-      return (
-         <div>Navbar here</div>
-      )
-   }
+	render() {
+		return (
+			<div>
+				<div id="navbar">
+					<div id="general-logo">
+						<img id="logo-hack" src={'img/logo-hack.png'} alt="" />
+						<img id="hack-wordmark" src={'img/hack-wordmark-darkcolor.png'} alt="" />
+					</div>
+
+					<div className="inner-menu desktop-only">
+						<div id="tab-container">
+							<span className="tab-style">Home</span>
+							<span className="tab-style">Events</span>
+							<span className="tab-style">Showcase</span>
+							<span className="tab-style">Team</span>
+						</div>
+					
+						<div className="tab-style" id="login-box">Login</div>
+					</div>
+
+					<div className="ham-menu mobile-only">
```

Comment:
You can add a React `onClick` handler here to replace your external JS functionality. And in that handler, set some state for conditional rendering.